lib/export-contact-history-s3.js

```diff
@@ -45,6 +45,10 @@ function exportContactHistory(scope) {
     tablePolicy.addActions(['honeycode:ListTableColumns', 'honeycode:BatchUpdateTableRows', 'honeycode:QueryTableRows']);
     tablePolicy.addResources([`arn:aws:honeycode:*:*:table:workbook/${environment.workbookId}/table/*`]);
     exportContactHistory.role.addToPrincipalPolicy(tablePolicy)
+    //Output the Bucket URL
+    new cdk.CfnOutput(scope, "Contact History Bucket Name", {
```

Comment:
This is great! it would be nice to have this information written in the manifest.json that we need to download for QuickSight integration, or create that file along with this output and use the --outputs-file option from CDK to write the output to a file.

Reply:
See comment above about the change to how the manifest file is now being handled. An AWS custom resource is used to upload the manifest file to the contacthistory S3 bucket and the URL to the file is used to create the dataset in QuickSight